### PR TITLE
Helm install check

### PR DESCRIPTION
**Pull request by kskzk**
Issue #, if available:  https://github.com/aws-controllers-k8s/community/issues/1436

Description of changes:
- Add helm version check function
- Add helm installation helper script
- Modify k8s_controller_gen_version_equals function and its comment

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by A-Hilaly — APPROVED** [review]
Neat :+1:  
Thank you @kskzk !

File: scripts/lib/common.sh [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 
 CONTROLLER_TOOLS_VERSION="v0.7.0"
+HELM_VERSION="v3.9"
```
Comment (A-Hilaly) [inline scripts/lib/common.sh:4]: Let's drop the version to `3.7` ? i guess 3.9 is not required.

Comment (A-Hilaly) [reply scripts/lib/common.sh:4]: Or maybe we need to update our prow jobs to use 3.9 helm binaries..

Comment (A-Hilaly) [reply scripts/lib/common.sh:4]: Helm is installed here https://github.com/aws-controllers-k8s/test-infra/blob/da4dad6c57818a0d4f87f8e2ac9e608612bfaff8/prow/jobs/images/Dockerfile.auto-generate-controllers and few other dockerfiles in the same directory

Comment (kskzk) [reply scripts/lib/common.sh:4]: Thanks for the review!
Which do you think is better? Do we drop the minimum required version to 3.7 for now? If so, I will modify the pull request.

Comment (A-Hilaly) [reply scripts/lib/common.sh:4]: I'm thinking that maybe we should start by dropping the version to 3.7 and dealing with the upgrade in a different PR

**Comment by A-Hilaly** [comment]
Neat! Thank you @kskzk!
/approve
/lgtm

**Pull request merged**